setup_bsc_node.sh

```diff
@@ -4,10 +4,8 @@ workspace=${basedir}
 source ${workspace}/.env
 source ${workspace}/utils.sh
 size=$((${BSC_CLUSTER_SIZE}))
-native_need_bls=false
 nodeurl="http://localhost:26657"
 replaceWhitelabelRelayer="0xb005741528b86F5952469d80A8614591E3c5B632"
-initConsensusStateBytes=$(${workspace}/bin/tool -height 1 -rpc ${nodeurl} -network-type 1)
```

Comment:
if we run `bash +x ./setup_bsc_node.sh native_init ` after
                `bash +x ./setup_bc_node.sh native_start` immediately
initConsensusStateBytes may contain an error, because bc has not startted up fully.